src/extension.ts

```diff
@@ -220,18 +221,16 @@ export async function resolveConnectionSpec(serverName: string): Promise<void> {
 }
 
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-export async function resolvePassword(serverSpec): Promise<void> {
+export async function resolvePassword(serverSpec, ignoreUnauthentiated = false): Promise<void> {
```

Comment:
Typo in local symbol name
```suggestion
export async function resolvePassword(serverSpec, ignoreUnauthenticated = false): Promise<void> {
```